packages/ui/src/routes/send/SendConfirmPage.tsx

```diff
@@ -808,8 +747,11 @@ const SendConfirmPage = () => {
                     style={{ maxHeight: "452px" }}
                 >
                     <AddressDisplay
-                        showingTheWholeAddress={showingTheWholeAddress}
-                        setShowingTheWholeAddress={setShowingTheWholeAddress}
+                        receivingAddress={history.location.state.address}
+                        selectedAccountName={
+                            history.location.state.name ??
+                            useAccountNameByAddress(receivingAddress)
```

Comment:
I would not put this hook invoke here at all. Can you move it where the other hooks declarations are?